### PR TITLE
LPS-201521 - Import/Export Object Folder Integration Tests

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPS-201521

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @G-Sa 
Could you review my changes? Thank you!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java [hunk 1]
```diff
@@ -56,88 +43,26 @@
...
-			WebKeys.THEME_DISPLAY, themeDisplay);
-
-		return mockLiferayResourceRequest;
-	}
-
-	private MockMultipartHttpServletRequest
-			_createMockMultipartHttpServletRequest(String fileName)
+	public void testExportImport(
+			String fileName, String expectedFileName,
+			String externalReferenceCode, String name)
```
Comment (G-Sa) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java:59]: We need to sort the parameter alphabetically, no?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java [hunk 2]
```diff
@@ -56,88 +43,26 @@
...
-			"custom_object_definition.json", "ImportedCustomObjectDefinition",
-			false);
-	}
+public abstract class BaseExportImportTestCase {
 
-	private MockLiferayPortletActionRequest
-			_createMockLiferayPortletActionRequest(
-				String externalReferenceCode, String fileName, String name)
+	public void testExportImport(
+			String fileName, String externalReferenceCode, String name)
```
Comment (G-Sa) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java:51]: We need to sort the parameters alphabetically

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectFolderExportImportTest.java [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+		testExportImport(
+			"object_folder_invalid_items.json", "empty_object_folder.json",
+			"EmptyObjectFolderERC", "EmptyObjectFolder");
+
+		_assertObjectFolder(
+			"EmptyObjectFolder", 0, Collections.emptyList(),
+			Collections.emptyList());
+
+		//  Import and export a linked object folder item that doesn't exist
```
Comment (G-Sa) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectFolderExportImportTest.java:78]: I think a better message would be `Import and export a folder with a linked object folder item that doesn't exist`. Its more verbose, but currently its sounding like we are importing and exporting a single object folder item

**Review by G-Sa — CHANGES_REQUESTED** [review]
Can you take a look and verify if these changes make sense?

**Comment by marcelabc** [comment]
ci:test:sf

**Review by G-Sa — APPROVED** [review]

**Comment by G-Sa** [comment]
ci:test:relevant

**Comment by G-Sa** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**